terraform/modules/vpc-hub/main.tf

```diff
@@ -288,6 +288,27 @@ resource "aws_route" "public-internet-gateway" {
   route_table_id         = aws_route_table.public.id
 }
 
+resource "aws_route" "public_mp_10-20-0-0--16" {
+  count                  = var.gateway == "nat" ? 1 : 0
+  destination_cidr_block = "10.20.0.0/16"
```

Comment:
Could we move the CIDR ranges to a local (probably outside the module) so that we can change them centrally if needed?  Eg have locals like:
```
core_cidr = 10.20.0.0/16
dev_test_cidr = 10.26.0.0/16
preprod_prod_cidr = 10.27.0.0/16
```
This way it's clear what range each resource is referring to and also if we have one central point for this range it's easy to update if we need to change them or rebuild in another region.

Reply:
out of scope for this